### PR TITLE
[EOS-17233] Use libfab format EP if libfabric is available

### PR DESCRIPTION
**Pull request by VinothVijayakumar-dev**
Changes related to rpc panic was resolved.
 - modified UT/ST to choose lnet as the default transport.
 - modified motr-server to choose libfab as transport if libfab is available.

### CODE REVIEW

File: cas/ut/client_ut.c [hunk 1]
```diff
@@ -81,7 +81,11 @@ enum { MAX_RPCS_IN_FLIGHT = 10 };
 static char *cas_startup_cmd[] = { "m0d", "-T", "linux",
                                 "-D", "cs_sdb", "-S", "cs_stob",
                                 "-A", "linuxstob:cs_addb_stob",
-                                "-e", "lnet:0@lo:12345:34:1",
+#ifdef ENABLE_LIBFAB
+                                "-e", "libfab:0@lo:12345:34:1",
+#else
+                                "-e", "libfab:0@lo:12345:34:1",
+#endif
```
Comment (huanghua78) [inline cas/ut/client_ut.c:88]: These two are identical?

File: scripts/install/usr/libexec/cortx-motr/motr-server [hunk 2]
```diff
@@ -224,10 +224,12 @@ m0_server()
     fi
 
     local service_ep
+    local motr_transport=$(m0_get_motr_transport)
+	m0_log "motr transport : $motr_transport"
```
Comment (huanghua78) [inline scripts/install/usr/libexec/cortx-motr/motr-server:228]: indent issue.

File: sns/cm/repair/ut/cp_common.c [hunk 3]
```diff
@@ -36,8 +36,13 @@ char      *sns_cm_ut_svc_ad[] = { "m0d", "-T", "AD",
                                   "-A", "linuxstob:sr_addb_stob",
 			          "-f", M0_UT_CONF_PROCESS,
 			          "-w", "10",
+#ifdef ENABLE_LIBFAB
+			          "-G", "libfab:0@lo:12345:34:1",
+                                  "-e", "libfab:0@lo:12345:34:1",
+#else
 			          "-G", "lnet:0@lo:12345:34:1",
                                   "-e", "lnet:0@lo:12345:34:1",
+#endif
```
Comment (huanghua78) [inline sns/cm/repair/ut/cp_common.c:45]: The original code has alignment issues. 
Please fix them.

**Comment by huanghua78** [comment]
@VinothVijayakumar-dev Can we update the title of this PR, e.g.
Use libfab format EP if libfabric is available

**Review by jamohite — APPROVED** [review]

**Review by upendrapatwardhan — APPROVED** [review]

**Pull request merged**